internal/bundler/bundler.go

```diff
@@ -2487,6 +2487,9 @@ type runtimeCacheKey struct {
 	unsupportedJSFeatures compat.JSFeature
 	minifySyntax          bool
 	minifyIdentifiers     bool
+	// TODO: Safe to use a pointer here? Is the target itself easier/safer?
+	// tsTargetValue	  *config.TSTarget.value?
+	tsTarget			  *config.TSTarget
```

Comment:
Please review/edit this line - I'll defer to you on whether an ES version number like `2015` is more appropriate than a pointer.